core/txpool/list.go

```diff
@@ -45,6 +45,7 @@ func (h *nonceHeap) Pop() interface{} {
 	old := *h
 	n := len(old)
 	x := old[n-1]
+	old[n-1] = 0
```

Comment:
Setting a `uint64` to `0` does not make any difference to the gc

Reply:
Slice tricks from the [Go wiki](https://github.com/golang/go/wiki/SliceTricks) states that you should set the item that is deleted to the zero value of T.

Reply:
Interesting!